app/src/pages/OnDeviceDisplay/ProtocolDetails/Hardware.tsx

```diff
@@ -113,22 +128,36 @@ export const Hardware = (props: { protocolId: string }): JSX.Element => {
           return (
             <TableRow key={id}>
               <TableDatum>
-                <StyledText
-                  as="p"
-                  color={COLORS.darkBlack100}
-                  paddingLeft={SPACING.spacing24}
-                >
-                  {i18n.format(getHardwareLocation(hardware, t), 'capitalize')}
-                </StyledText>
+                <Flex paddingLeft={SPACING.spacing24}>
+                  {hardware.hardwareType === 'module' ? (
+                    <LocationIcon slotName={hardware.slot} />
+                  ) : (
+                    <StyledText
+                      as="p"
+                      color={COLORS.darkBlack100}
+                      fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+                    >
+                      {i18n.format(
+                        getHardwareLocation(hardware, t),
+                        'titleCase'
+                      )}
+                    </StyledText>
+                  )}
+                </Flex>
               </TableDatum>
               <TableDatum>
-                <StyledText
-                  as="p"
-                  color={COLORS.darkBlack100}
-                  paddingLeft={SPACING.spacing24}
-                >
-                  {getHardwareName(hardware)}
-                </StyledText>
+                <Flex paddingLeft={SPACING.spacing24}>
+                  {hardware.hardwareType === 'module' &&
+                    getModuleIcon(hardware) !== null && (
```

Comment:
```suggestion
                    getModuleIcon(hardware) != null && (
```
or 

change getModuleIcon's return type `IconName | null`

Reply:
Have to keep the null to keep TS happy, but did change the inequality check.